core-bundle/src/Resources/contao/drivers/DC_Table.php

```diff
@@ -6331,6 +6331,11 @@ public function initPicker(PickerInterface $picker)
 
 		return $attributes;
 	}
+
+	private function addTableClass(string $table)
```

Comment:
```suggestion
	private function getTableClass(string $table)
```

It doesn't really add anything, does it? ;)